meilisearch_python_sdk/index.py

```diff
@@ -1307,7 +1316,16 @@ async def get_document(self, document_id: str) -> JsonDict:
             >>>     index = client.index("movies")
             >>>     document = await index.get_document("1234")
         """
-        response = await self._http_requests.get(f"{self._documents_url}/{document_id}")
+        parameters: JsonDict = {}
+
+        if fields:
+            parameters["fields"] = fields
```

Comment:
This needs to be a comma seperated list of values when sent.

Note to self, this probably means there is a bug in get_documents. I'll chack that and open a seperate issue if so.

```suggestion
            parameters["fields"] = ",".join(fields)
```